core/geometry/src/clipping/ConvexClipPlaneSet.ts

```diff
@@ -761,10 +781,16 @@ export class ConvexClipPlaneSet implements Clipper, PolygonClipper {
    * @param zHigh high z value.  The plane clips out points with z above this.
    */
   public addZClipPlanes(invisible: boolean, zLow?: number, zHigh?: number) {
-    if (zLow !== undefined)
-      this._planes.push(ClipPlane.createNormalAndDistance(Vector3d.create(0, 0, 1), zLow, invisible)!);
-    if (zHigh !== undefined)
-      this._planes.push(ClipPlane.createNormalAndDistance(Vector3d.create(0, 0, -1), -zHigh, invisible)!);
+    if (zLow !== undefined) {
```

Comment:
If you combine the if statements above, don't forget these.

Reply:
Sorry I dont understand your comment.